torch/fx/experimental/GraphManipulation.py

```diff
@@ -121,6 +121,21 @@ def serialize_weight(tensor: torch.Tensor) -> Dict:
     return weight
 
 
+def serialize_leaf_module(
+    mod: torch.nn.Module, weights: Dict, name_prefix: str
+) -> Dict:
+    parameters = {}  # type: Dict[str, Any]
+    parameters["name"] = type(mod).__name__
+    for name, buffer in mod.named_buffers():
+        weights[f"{name_prefix}.{name}"] = serialize_weight(buffer)
+    for name, parameter in mod.named_parameters():
+        weights[f"{name_prefix}.{name}"] = serialize_weight(parameter)
+    if isinstance(mod.__constants__, List):
+        for constant in mod.__constants__:
+            parameters[constant] = getattr(mod, constant)
```

Comment:
Would it be possible to make the value a string, like `str(getattr(mod, constant))`? This would help with the processing because constant like `kernel_size` could be an integer or a tuple.